posthog/models/async_deletion/delete.py

```diff
@@ -22,8 +22,37 @@
 
 
 def run_event_table_deletions():
-    queued_deletions = list(AsyncDeletion.objects.filter(delete_verified_at__isnull=True))
-    process_found_event_table_deletions(queued_deletions)
+    queued_event_deletions = AsyncDeletion.objects.filter(
+        delete_verified_at__isnull=True, deletion_type__in=[DeletionType.Team, DeletionType.Person, DeletionType.Group]
+    )
+    process_found_event_table_deletions(list(queued_event_deletions))
+
+    queued_cohortpeople_deletions = AsyncDeletion.objects.filter(
+        delete_verified_at__isnull=True, deletion_type__in=[DeletionType.Cohort_full, DeletionType.Cohort_stale]
+    )
+    process_found_cohort_table_deletions(list(queued_cohortpeople_deletions))
+
+
+def process_found_cohort_table_deletions(deletions: List[AsyncDeletion]):
+    if len(deletions) == 0:
+        logger.debug("No AsyncDeletion for cohorts to perform")
+        return
+
+    logger.info(
+        "Starting AsyncDeletion on `cohortpeople` table in ClickHouse",
+        {"count": len(deletions), "team_ids": list(set(row.team_id for row in deletions))},
+    )
+
+    conditions, args = _conditions(deletions)
+
+    sync_execute(
+        f"""
+        ALTER TABLE cohortpeople
+        ON CLUSTER '{CLICKHOUSE_CLUSTER}'
```

Comment:
We don't need on cluster here since cohortpeople is a replicated, but not sharded table